api/server/server.go

```diff
@@ -39,7 +39,8 @@ import (
 	"github.com/docker/docker/utils"
 )
 
-type ServerConfig struct {
+//APIServerConfig - provides the configuration for the API server
```

Comment:
@brahmaroutu you don't need to add the `` - ``. I think the "look" of godoc will be a little weird.